app/lib/server/server.go

```diff
@@ -0,0 +1,499 @@
+package server
+
```

Comment:
is this just renaming the file `app/cothorityd/config.go` or are there any other changes?